src/Commands/SearchApiSolrCommands.php

```diff
@@ -100,10 +100,13 @@ public function installMissingFieldtypes() {
    * @throws \ZipStream\Exception\OverflowException
    */
   public function getServerConfig($server_id, $file_name = NULL, $solr_version = NULL, array $options = []) {
-    if (!$options['pipe'] && ($file_name === NULL)) {
-      throw new ConsoleException('Required argument missing ("file_name"), and no --pipe option specified.');
+    if (isset($options['pipe'])) {
```

Comment:
This is wrong. You turned off the check if a file name has been provided instead of using a pipe.